variables.tf

```diff
@@ -61,17 +61,85 @@ variable "deployment" {
   nullable    = false
 }
 
+variable "dynamic_throttling_enabled" {
+  type        = bool
+  default     = null
+  description = "Determines whether or not dynamic throttling is enabled. If set to true, dynamic throttling will be enabled. If set to false, dynamic throttling will not be enabled."
+}
+
 variable "environment" {
   type        = string
   default     = ""
   description = "Environment of the application. A corresponding tag would be created on the created resources if `var.default_tags_enabled` is `true`."
 }
 
-# tflint-ignore: terraform_unused_declarations
-variable "network_acls_default_action" {
-  type        = string
-  default     = "Deny"
-  description = "Azure OpenAI account network ACLs default action."
+variable "customer_managed_key" {
```

Comment:
You want to indicate that the variable is sensitive so that the value will not be printed out when running Terraform:
https://developer.hashicorp.com/terraform/tutorials/configuration-language/sensitive-variables

`sensitive = True`